ocaml/sdk-gen/go/templates/Enum.mustache

```diff
@@ -6,6 +6,8 @@ const (
 	//{{#doc}} {{.}}{{/doc}}
 	{{name}} {{type}} = "{{value}}"
 {{/values}}
+	// The value does not belong to this enumeration
+	{{name}}Unrecognized {{name}} = "unrecognized"
```

Comment:
can't it just be the following or will this cause issues?

```go
	Unrecognized {{name}} = "unrecognized"
```

Reply:
point being that I don't thin we need a prefix here 🤔

Reply:
It a good point but unfortunately we can't move the prefix here in Go, like you can see:
![image](https://github.com/user-attachments/assets/50c4dd7f-e38a-4eb0-b591-2c6676fcd581)
![image](https://github.com/user-attachments/assets/4f85c426-691d-49f4-b9e1-a773d5de24ea)

Reply:
ah! gotcha. makes sense, thanks for clarifying